### PR TITLE
Short explanation of the sockets trouble in the docs.

### PR DESCRIPTION
**Pull request by stychos**
We have a growing number of issues related to the sockets connection trouble. Maybe this line in the docs can help a little?

What does this implement/fix? Explain your changes.
---------------------------------------------------
nothing

Does this close any currently open issues?
---------------------------------------－
no

### CODE REVIEW

File: docs/get-started/local-connection.md [hunk 1]
```diff
@@ -19,11 +19,7 @@ If you are not sure if the MySQL server is running, open _Activity Viewer_ (from
 
 #### Connecting via a socket connection
 
-Open Sequel Ace. Choose a _Socket_ Connection. You must only specify the username and password (if any). Most MySQL installations use the default username root and a blank password.
-
-If you leave the socket field empty, Sequel Ace will try several common socket file locations. If Sequel Ace can't find your socket file, or if you have multiple MySQL servers running on your computer, you must enter the location of the socket file.
-
-**Note**: the popular MAMP package uses root as default password. See [Connecting to MAMP or XAMPP.](mamp-xampp.html "Connecting to MAMP or XAMPP")
+Unfortunately, due to sandboxing nature, Sequel Ace is not allowed to connect to the sockets which are out of the Sandbox. As a workaround, you can create a socket in `~/Library/Containers/com.sequel-ace.sequel-ace/Data` and connect to it.
```
Comment (Kaspik) [inline docs/get-started/local-connection.md:22]: Should we also add the command to create the socket?

Comment (stychos) [reply docs/get-started/local-connection.md:22]: Ahh, you right!

Comment (stychos) [reply docs/get-started/local-connection.md]: Added configuration sample.

**Comment by Jason-Morcos** [comment]
@stychos I just realized we may also want to add some of this info to the Getting Connected FAQ directly! It's a little confusing. https://sequel-ace.com/get-started/

**Pull request merged**